Makefile

```diff
@@ -39,6 +39,7 @@ GVT ?= $(GOPATH)/bin/gvt$(BIN_ARCH)
 GOVC ?= $(GOPATH)/bin/govc$(BIN_ARCH)
 GAS ?= $(GOPATH)/bin/gas$(BIN_ARCH)
 MISSPELL ?= $(GOPATH)/bin/misspell$(BIN_ARCH)
+DOCKER_BUILD := $(BASE_DIR)/infra/build-image/docker-iso-build.sh
```

Comment:
The cgroup switch is to prevent the build from starting a docker image.
If the build is running in CI, we don't want to start docker in docker, we just want to run the make in the ci container itself.

If run on a containerVM, the `cgroups` check will faile, and the script will try to start a docker container as it would on any other VM. 
I ignored the "building in a containerVM use case", but we can investigate it at a later time.